src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,27 @@
+@using OrchardCore.Forms.ViewModels;
+@model ValidationRulePartEditViewModel
+
+<div class="form-group" >
+    <label asp-for="Type">@T["Validate Type"]</label>
+    <select asp-for="Type" class="form-control content-preview-select">
+        <option value="none">@T["None"]</option>
+        <option value="contains">@T["Contains"]</option>
+        <option value="isDate">@T["IsDate"]</option>
+        <option value="isDecimal">@T["IsDecimal"]</option>
+        <option value="isEmail">@T["IsEmail"]</option>
+        <option value="isEmpty">@T["IsEmpty"]</option>
+        <option value="isInt">@T["IsInt"]</option>
+        <option value="isIP">@T["IsIP"]</option>
+        <option value="isURL">@T["IsURL"]</option>
+        <option value="matches">@T["Matches"]</option>
+
+    </select>
+    <span class="hint">@T["The validate type."]</span>
+</div>
+
+<div class="form-group" >
+    <label asp-for="Option">@T["Validate Options"]</label>
+    <input asp-for="Option" type="text" class="form-control content-preview-text" />
+    <span class="hint">@T["The text to check the string. For detailed rules, please see https://github.com/validatorjs/validator.js"]</span>
```

Comment:
Could you add some docs on using this? https://docs.orchardcore.net/en/dev/docs/topics/docs-contributions/

Also, make the GitHub URL a link.